roottest/root/treeformula/array/CMakeLists.txt

```diff
@@ -1,7 +1,87 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST(LABELS longtest)
+ROOTTEST_COMPILE_MACRO(Data.cxx
+                       FIXTURES_SETUP root-treeformula-array-Data_cxx-fixture)
+
+ROOTTEST_ADD_TEST(filltree
+                  MACRO fillTree.cxx
```

Comment:
Side note: in previous version fillTree.cxx was used compiled, however it does not appear to be important to the test.